src/Domain.cpp

```diff
@@ -291,14 +291,19 @@ void Domain::calculateGlobalValues(
 				}
 			} /*_OPENMP*/
 
-			/* FIXME: Unnamed constant 3960... */
-			if(3960 >= _universalSelectiveThermostatCounter)
-			{
+			int explosionReappearanceLimit = 4000;
+			int explosionVanishGracePeriod = 40;
```

Comment:
I guess these are completely arbitrary? ;) If so please note this in the code. If not explain why these values.

Reply:
```suggestion
			// arbitrary values set by one of the thermodynamic guys :)
			int explosionReappearanceLimit = 4000;
			int explosionVanishGracePeriod = 40;
```
is this fine for you?

Reply:
lol yes